pandas/io/tests/test_excel.py

```diff
@@ -1083,6 +1089,31 @@ def test_to_excel_multiindex(self):
                               parse_dates=False)
             tm.assert_frame_equal(frame, df)
             self.assertEqual(frame.index.names, df.index.names)
+            
+    def test_to_excel_multiindex_cols(self):
+        _skip_if_no_xlrd()
+        if not self.merge_cells:
+            raise nose.SkipTest('Skip tests MI on cols with no merging.')
+
+
+        frame = self.frame
+        arrays = np.arange(len(frame.index) * 2).reshape(2, -1)
+        new_index = MultiIndex.from_arrays(arrays,
+                                           names=['first', 'second'])
+        frame.index = new_index
+
+        new_cols_index = MultiIndex.from_tuples([(40,1),(40,2),(50,1),(50,2)])
+        frame.columns = new_cols_index
+
+        with ensure_clean(self.ext) as path:
+             # round trip
+            frame.to_excel(path, 'test1', merge_cells=self.merge_cells)
+            reader = ExcelFile(path)
+            df = read_excel(reader, 'test1', header=[0,1],
+                             index_col=[0, 1],
+                              parse_dates=False)
+            tm.assert_frame_equal(frame, df)
+            self.assertEqual(frame.index.names, df.index.names)
```

Comment:
I deleted that, but also deleted it from the test above, which is where I copied the code from.